jkrimporter/providers/db/services/buildings.py

```diff
@@ -38,20 +38,20 @@
 
 def minimum_distance_of_buildings(buildings: List[Rakennus]) -> float:
```

Comment:
Miten tämä vaikuttaa buildings.py rivin 234 toimintaan kun tämän palauttama arvo on kääntynyt päinvastaiseksi? Tulisiko nimi vaihtaa?
Tarkoittaako tämä nyt sitä että jos sille annetaan kolmen rakennuksen lista joista kaksi ovat lähekkäin ja kolmas liian kaukana niin ne kaksi ei yhdistykkään?

Reply:
Niin ja eikös tämä nyt palauta isoimman etäisyyden rakennusten välillä jolloin jos ne oli vaikka 250m etäisyydellä toisistaan, mutta kauimmat nurkat on 310 metrissä toisistaan, niin sithän ne ei mee samaan kohteeseen?